tests/v3_api/common.py

```diff
@@ -313,7 +321,8 @@ def validate_ingress(p_client, cluster, workload, host, path):
     pods = p_client.list_pod(workloadId=workload.id)
     target_name_list=[]
     for pod in pods:
-        target_name_list.append(pod.name.encode('UTF-8'))
+        target_name_list.append(pod.name)
+    print target_name_list
```

Comment:
why do we print the list here? if its for log purposes can we provide more information like `target name list: [a,b,c]`